### PR TITLE
chore/test-operance-data-column

### PR DESCRIPTION
**Pull request by zxcvbnmmohd**

### CODE REVIEW

File: test/operance_data_column_test.dart [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+// 🌎 Project imports:
+import 'package:operance_datatable/src/operance_data_column.dart';
+import 'package:operance_datatable/src/operance_data_column_width.dart';
+
+void main() {
+  group('Given an OperanceDataColumn instance', () {
+    group('When all required parameters are provided', () {
+      test('Then it should create an instance with default values', () {
+        final column = OperanceDataColumn<String>(
```
Comment (fjyeo) [inline test/operance_data_column_test.dart:15]: could add type annotations here

File: test/operance_data_column_test.dart [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+          name: 'name',
+          columnHeader: const Text('Header'),
+          cellBuilder: (context, item) => Text(item),
+        );
+
+        expect(column.name, 'name');
+        expect(column.columnHeader, isA<Text>());
+        expect(column.cellBuilder, isA<Function>());
```
Comment (fjyeo) [inline test/operance_data_column_test.dart:23]: could test the cell builder and getsearchablevalue function

**Review by fjyeo — COMMENTED** [review]
Great work just a few thoughts

File: test/operance_data_column_test.dart (hunk 1 above)
Comment (zxcvbnmmohd) [reply test/operance_data_column_test.dart:15]: what do you mean?

**Pull request closed without merging**